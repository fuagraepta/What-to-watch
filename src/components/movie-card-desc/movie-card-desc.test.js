import React from 'react';
import renderer from 'react-test-renderer';

import MovieCardDesc from './movie-card-desc.jsx';

const mockElement = <div></div>;

describe(`The component is rendered correctly`, () =>{
  it(`Movie card description correctly renders with transferred mock data`, () =>{
    const movieCardDesc = renderer
    .create(<MovieCardDesc
      title = {`Johnny English`}
      genre = {`Comedies`}
      releaseDate = {2014}
    >
      {mockElement}
    </MovieCardDesc>
    )
    .toJSON();

    expect(movieCardDesc).toMatchSnapshot();
  });
});
