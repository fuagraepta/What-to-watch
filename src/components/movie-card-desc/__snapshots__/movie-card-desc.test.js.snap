// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The component is rendered correctly Movie card description correctly renders with transferred mock data 1`] = `
<div
  className="movie-card__desc"
>
  <h2
    className="movie-card__title"
  >
    Johnny English
  </h2>
  <p
    className="movie-card__meta"
  >
    <span
      className="movie-card__genre"
    >
      Comedies
    </span>
    <span
      className="movie-card__year"
    >
      2014
    </span>
  </p>
  <div />
</div>
`;
